Parse/src/main/java/com/parse/Parse.java

```diff
@@ -158,72 +152,73 @@ public Builder server(String server) {
       }
 
       /**
-       * Add a {@link ParseNetworkInterceptor}.
-       *
-       * @param interceptor The interceptor to add.
-       * @return The same builder, for easy chaining.
-       */
-      public Builder addNetworkInterceptor(ParseNetworkInterceptor interceptor) {
-        if (interceptors == null) {
-          interceptors = new ArrayList<>();
-        }
-        interceptors.add(interceptor);
-        return this;
-      }
-
-      /**
        * Enable pinning in your application. This must be called before your application can use
        * pinning.
+       *
        * @return The same builder, for easy chaining.
        */
       public Builder enableLocalDataStore() {
         localDataStoreEnabled = true;
         return this;
       }
 
-      /* package for tests */ Builder setNetworkInterceptors(Collection<ParseNetworkInterceptor> interceptors) {
```

Comment:
@flovilmart I get your concern here, but OkHttp is pretty much standard for a library/application that is using HTTP. We are deprecating that interceptors library in favor of just having people use OkHttp interceptors for logging, which allows for more control and selection for interceptors (including logging ones) such as https://github.com/jgilfelt/chuck. Most people will probably be more familiar with setting up an OkHttp interceptor than a Parse specific one. Also, relying on a tried and true third party library is favorable in my opinion over having three different HTTP implementations to maintain/debug.